### ISSUE TITLE
Updating Setting > Support options

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-android#2056]
@colourfreak this is how it looks like in iOS:

<img width="200" alt="Screenshot 2021-09-28 at 2 54 06 PM" src="https://user-images.githubusercontent.com/56189/135037810-ee5fc6ed-d799-48b0-b3af-032ff491eb33.png">

Can you help verify if that's the correct list? We have this https://app.zeplin.io/project/5d0356be2f25b319dcc5944b/screen/5dfc1f52a3e2aa1c676e66d4 in https://github.com/AlphaWallet/alpha-wallet-ios/issues/2982

**Comment by colourfreak** [comment]
This is correct. No Facebook, no Blog, no Reddit. This is confirmed with Victor.

**Comment by JamesSmartCell** [comment]
@colourfreak should we have a github link here? To make it easy to submit issues if the user knows github and so they know we're open source.

**Comment by colourfreak** [comment]
@hboon Shall we add the same for iOS?
